torchmultimodal/modules/layers/transformer_decoder.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+from torch import nn
+from torch.utils.checkpoint import checkpoint
+
+from torchmultimodal.modules.layers.attention import FullAttention, MultiHeadAttention
+from torchmultimodal.modules.layers.mlp import MLP
+
+
+class SiLU(nn.Module):
```

Comment:
Is this actually SiLU though? Pytorch already has SiLU as x * sigma(x). I see why you need this one but would consider naming it differently

Reply:
This is activation used by the transformer decoder layer

Reply:
It's different from pytorch SiLU by a multiplier of 1.702 to the input tensor. Amended docstring to highlight.